kernel/Dockerfile

```diff
@@ -26,8 +25,7 @@ RUN apk add \
     tar \
     xz \
     xz-dev \
-    zlib-dev \
-    || true
```

Comment:
If we keep them sorted then the chances of something coming after `zlib` are at least slim...

Or the list could end
```
    zlib-dev \
    # end of list
```
although I'm not sure that is nicer than the occasional whitespace-fu on update.